docs/folder-structure.md

```diff
@@ -152,6 +153,65 @@ Flake outputs:
 
 > Depending on the system type returned, the flake outputs will be the same as detailed for NixOS or Darwin above.
 
+### `homes/<name>/(default.nix|home.nix)`
+
+Each folder contains a standalone home-manager configuration:
+
+#### `home.nix`
+
+Evaluates to a standalone home-manager configuration.
+
+Additonal values passed:
+
+* `inputs` maps to the current flake inputs.
+* `flake` maps to `inputs.self`.
+* `pkgs` an instance of nixpkgs, see [configuration](configuration.md) on how it's configured.* `perSystem`: contains the packages of all the inputs, filtered per sysstem.
+    Eg: `perSystem.nixos-anywhere.default` is a shorthand for `inputs.nixos-anywhere.packages.<system>.default`.
+
+Flake outputs:
+
+* `homeConfigurations."<name>@<system>"`
```

Comment:
In the current home-manager implementation, the @ sign is used for `<user>@<hostname>`, which is handy if you want a different profile on a specific host (eg: a server profile with no GUI apps vs a desktop profile).

Do you have any ideas on how to handle that case?

Reply:
I'm not really sure how to handle this tbh.

A couple of ugly ideas off the top of my head:
- subfolders for machine architecture, removing the need for `user@architecture`
- `user@hostname::architecture` -  I don't know if there are conventions for hostnames, but I can see a machine out there having `::` in the name. I can't think of a better separator.


Any suggestions?